CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ---
 
+## [6.1.0] - 2024-11-29
+
+### Added
+
+- ([#208](https://github.com/primait/amqpx/pull/208)) Introduce the graceful termination using the application signal handler to cancel the consumers on draining phase and close the channels on stopping phase.
```

Comment:
Not to be too nitpick, but technically the PR introduces the possibility of configuring a signal handler which can be used for graceful termination and so on, right?

If so, I would rephrase this a bit so expectations are clearer